big_tests/tests/mongoose_user_stats_SUITE.erl

```diff
@@ -0,0 +1,158 @@
+-module(mongoose_user_stats_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/ms_transform.hrl").
+
+-define(COWBOY_DUMMY_HANDLER_MODULE, my_cowboy_mecked_module).
+-define(SERVER_URL, "http://localhost:8765").
+-define(ETS_TABLE, qs).
+
+-record(event, {
+    el = "",
+    cid = "",
+    ec = "",
+    ea = ""
+    }).
+
+-compile(export_all).
+
+-import(distributed_helper, [mim/0,
+                             require_rpc_nodes/1,
+                             rpc/4]).
+
+%%--------------------------------------------------------------------
+%% Suite configuration
+%%--------------------------------------------------------------------
+
+suite() ->
+    require_rpc_nodes([mim]).
+
+all() ->
+    [
+        number_of_hosts_is_reported_to_google_analytics_when_mim_starts
+    ].
+
+groups() ->
+    [].
+
+%%--------------------------------------------------------------------
+%% Init & teardown
+%%--------------------------------------------------------------------
+
+-define(APPS, [inets, crypto, ssl, fusco, ranch, cowlib, cowboy]).
+
+init_per_suite(Config) ->
+    [ {ok, _} = application:ensure_all_started(App) || App <- ?APPS ],
+    create_dummy_cowboy_handler(),
+    start_cowboy(),
+    Config.
+
+end_per_suite(Config) ->
+    stop_cowboy(),
+    remove_dummy_cowboy_handler(),
+    Config.
+
+init_per_group(_GroupName, Config) ->
+    Config.
+
+end_per_group(_GroupName, Config) ->
+    Config.
+
+init_per_testcase(_CaseName, Config) ->
+    ets:new(?ETS_TABLE, [duplicate_bag, named_table, public]),
+    Args = [google_analytics_url, "http://localhost:8765?dummy=arg"],
+    {atomic, ok} = mongoose_helper:successful_rpc(ejabberd_config, add_local_option, Args),
+    Config.
+
+end_per_testcase(_CaseName, Config) ->
+    Config.
+
+%%--------------------------------------------------------------------
+%% Tests
+%%--------------------------------------------------------------------
+
+
+number_of_hosts_is_reported_to_google_analytics_when_mim_starts(_Config) ->
```

Comment:
I think the name of the test case doesn't reflect that it also check if modules are reported correctly.

Reply:
I will rename it